### ISSUE TITLE
Not all HP or PW is displayed above 9999

### ISSUE THREAD

**Issue by Labtop-215** [NetHack/NetHack#206]
If my maximum HP or PW (or both) goes above 9999, the display only shows 9999 (9999).  From my understanding of the source code from here:

https://nethackwiki.com/wiki/Source:NetHack_3.6.1/src/botl.c

Various min(x,9999)'s are to avoid having excessive values
      * violate the field width assumptions in botl.h and should not
      * impact normal play.  Particularly 64-bit long for gold which
      * could require many more digits if someone figures out a way
      * to get and carry a really large (or negative) amount of it.
      * Turn counter is also long, but we'll risk that.

I understand that this is supposed to help create a consistent interface display, however I now have no way of knowing beyond trial and error how high my maximum power actually is.

I have verified that my actual power is higher than my maximum power by casting spells to bring down my current value to within the readable amount.  In previous versions of NetHack (I've been playing since 3.4.3 if that helps) I was able to see my maximum power go up beyond 9999.

I've tried using a blessed stethoscope on myself, a blessed wand of probing, a blessed potion of enlightenment, and a blessed wand of enlightenment on myself, to no avail.  None of these show my actual maximum amount of power.

Is there something that I am missing that I could do to actually see how high my maximum power is?

**Comment by copperwater** [comment]
Yes, the #attributes extended command (or Ctrl-X) shows a "Basics" section which prints your current and maximum HP and Pw, and does this correctly for values over 9999. No special items are needed.

For the bottom status line, 9999 is a reasonable maximum in my opinion. If you have more than that in Pw, you can do anything magical anyway.

**Comment by nhmall** [comment]
This issue was opened for the 3.6.1 version of NetHack that is no longer being actively maintained. 

Please try the NetHack-5.0.0 release. If you are still encountering an issue in the most-recent version, please open an issue and specify the NetHack version in the opening comment.
Thank you.
